RELEASE_NOTES.md

```diff
@@ -1,3 +1,10 @@
+# Version 0.5.13
```

Comment:
version 0.6.0 - this is not a bug fix, it is a feature addition (new parser)

Reply:
Done